### PR TITLE
feat: add whyWeDidIt section

### PR DESCRIPTION
**Pull request by Charbel-t**

### CODE REVIEW

File: src/Components/WhyWeDidItSection/WhyWeDidItSection.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './WhyWeDidItSection.css';
+import { Typography } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+function WhyWeDidItSection({ imgSrc, title, text }) {
```
Comment (osamaakb) [inline src/Components/WhyWeDidItSection/WhyWeDidItSection.js:7]: if the text, title and image doesn't change you don't need them as props just hardcode them in the component itself

File: src/Components/WhyWeDidItSection/WhyWeDidItSection.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className="align">
+      <img src={imgSrc} className="icon-size" />
```
Comment (osamaakb) [inline src/Components/WhyWeDidItSection/WhyWeDidItSection.js:10]: is this an icon? cause the css class name says it's an icon

**Review by osamaakb — CHANGES_REQUESTED** [review]

File: src/Components/WhyWeDidItSection/WhyWeDidItSection.js (hunk 1 above)
Comment (Charbel-t) [reply src/Components/WhyWeDidItSection/WhyWeDidItSection.js:7]: we'll call this component 3 times and each time different props similar to this section in this website
(Im tryna attach a screen-shot to show u what i meant but it's not uploading dunno why, i'll send it on Slack)

File: src/Components/WhyWeDidItSection/WhyWeDidItSection.js (hunk 2 above)
Comment (Charbel-t) [reply src/Components/WhyWeDidItSection/WhyWeDidItSection.js:10]: Yes it is, but as u told us yesterday bcz we were facing a problem with the svg file, to download it and insert it as an image, the className is just describing it lol, but in reality it's an img of that icon :)

**Review by osamaakb — APPROVED** [review]

**Pull request merged**